### ISSUE TITLE
Rules are required when defining a accesspolicy

### ISSUE THREAD

**Issue by jhrv** [nais/naiserator#109]
When you define a accessPolicy you are required to have a `spec.accessPolicy.outbound.rules` defined. 

Example of nais.yaml that should be valid:

```
spec:
  accessPolicy:
    outbound:
      external:
        - host: myexternalhost.no
```

**Comment by linemos** [comment]
If defined `spec.accessPolicy` you have to define at least one of `spec.accessPolicy.inbound.rules`, `spec.accessPolicy.outbound.rules` or `spec.accessPolicy.outbound.external`, right?

**Comment by jhrv** [comment]
fixed by #127
